internal/cli/tenants.go

```diff
@@ -203,6 +203,9 @@ func addTenantCmd(cli *cli) *cobra.Command {
 		},
 	}
 
+	tenantClientID.RegisterString(cmd, &inputs.ClientID, "")
+	tenantClientSecret.RegisterString(cmd, &inputs.ClientSecret, "")
```

Comment:
The `auth0 tenants add` command never respected the usage of of client ID and client secret flags. This fixes that.